roles/netbootxyz/templates/menu/k3os.ipxe.j2

```diff
@@ -1,24 +1,76 @@
 #!ipxe
 
-# k3OS
+# k3OS by Rancher
 # https://k3os.io/
+# https://github.com/rancher/k3os#configuration-reference
 
 goto ${menu} ||
 
 :k3os
-set os {{ releases.k3os.name }}
```

Comment:
Needs to be set as it pulls it from the defaults yaml in case there are future changes.